src/layout.ts

```diff
@@ -85,7 +85,8 @@ export async function prepareLayout(memento: vscode.Memento) {
         return;
       }
       if (isCodespaceActivation) {
-        child_process.execSync("touch $HOME/.config/vscode-dev-containers/first-run-notice-already-displayed");
+        // Commented out for now
+        // child_process.execSync("touch $HOME/.config/vscode-dev-containers/first-run-notice-already-displayed");
```

Comment:
@lostintangent we commented out in our fork temporarily for now, only because this directory might not exist by default in some codespaces, so thought we'd touch base on such first!